api/v1alpha/src/accounts/dataManager.js

```diff
@@ -313,9 +346,170 @@ async function deleteAccount(projectId, accountId, data) {
     }
 }
 
+/**
+ * @param  {} projectId
+ * @param  {} host
+ * @param  {} token
+ */
+async function register(projectId, host, token) {
+    // https://cloud.google.com/marketplace/docs/partners/integrated-saas/frontend-integration#verify-jwt
+    const jwt = require('jsonwebtoken');
+
+    /*
+        1. Verify that the JWT signature is using the public key from Google.
+        2. Verify that the JWT has not expired, by checking the exp claim.
+        3. Verify that aud claim is the correct domain for your solution.
+        4. Verify that the iss claim is https://www.googleapis.com/robot/v1/metadata/x509/cloud-commerce-partner@system.gserviceaccount.com
+        5. Verify that sub is not empty.
+    */
```

Comment:
Can we assume that the JWT signature has been validated from the Istio ingress gateway now and just decode the JWT for the payload?